src/course-search/config/index.js

```diff
@@ -1,13 +1,13 @@
-import qs from 'query-string';
-
 import {
   FEATURE_ENROLL_WITH_CODES,
   FEATURE_LANGUAGE_FACET,
   FEATURE_PROGRAM_TITLES_FACET,
 } from '../../constants';
+import { paramsToObject } from '../data/utils';
 
 const hasFeatureFlagEnabled = (featureFlag) => {
```

Comment:
esp. given we are changing how it's implemented, worth moving `hasFeatureFlagEnabled` into a util function and having a test I would suggest, it's kind of an important function and will help us avoid having to manually test this